charts/openebs/templates/kyverno/allow-host-namespaces.yaml

```diff
@@ -0,0 +1,32 @@
+{{- if Values.rbac.kyverno Enabled }}
```

Comment:
nit: this should be `{{- if .Values.rbac.kyvernoEnabled }}` , similarly in other places as well.